bigbluebutton-html5/imports/ui/components/nav-bar/recording-indicator/styles.scss

```diff
@@ -21,3 +22,19 @@
     background-color: $color-danger;
   }
 }
+
+.divider {
```

Comment:
Why the CSS for the new divider differ from the old one? Cant you just add a `border-left` on the outer `<div>`?